rest/diagnostic_doc_api.go

```diff
@@ -39,3 +56,64 @@ func (h *handler) handleGetDocChannels() error {
 	h.writeJSON(resp)
 	return nil
 }
+
+// HTTP handler for running a document through the sync function and returning the results
+// body only provided, the sync function will run with no oldDoc provided
+// body and doc ID provided, the sync function will run using the current revision in the bucket as oldDoc
+// docid only provided, if the doc exists in the bucket, sync fn results will be returned, otherwise it will error
```

Comment:
```suggestion
// docid only provided, the sync function will run using the current revision in the bucket as doc
// If docid is specified and the document does not exist in the bucket, it will return error
```